cmd/utils/flags.go

```diff
@@ -761,14 +761,24 @@ var (
 	}
 	ChainDataFetcherKafkaReplicasFlag = cli.Int64Flag{
 		Name:  "chaindatafetcher.kafka.replicas",
-		Usage: "Kafka replicas",
+		Usage: "Kafka partition replication factor",
 		Value: kafka.DefaultReplicas,
 	}
-	ChainDataFetcherKafkaPartitionsFlag = cli.Int64Flag{
+	ChainDataFetcherKafkaPartitionsFlag = cli.IntFlag{
 		Name:  "chaindatafetcher.kafka.partitions",
-		Usage: "Kafka broker partition number",
+		Usage: "The number of partitions in a topic",
 		Value: kafka.DefaultPartitions,
 	}
+	ChainDataFetcherKafkaMaxMessageBytesFlag = cli.Int64Flag{
+		Name:  "chaindatafetcher.kafka.max.message.bytes",
+		Usage: "Kafka producer max size of a message",
```

Comment:
This expression is a bit confusing.. Could you make it more clear? Or what about "The max message size allowed by Kafka" ?

Reply:
Applied 62e4d31c00f701df2dd3e49071ab0f0e2a4aa1f5. Thanks.